### PR TITLE
entries: lik-sang compo

### PR DESCRIPTION
**Pull request by dag7dev**
## Description
Roms have been tested one-by-one by hand.

Newer infos have been integrated with the already existent entries.

Happy liksang-compo-2000!~

### CODE REVIEW

File: entries/capman/game.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (avivace) [inline entries/capman/game.json:1]: Isn't this a duplicate? (I can see you also edited `Capman` with capital C)

**Review by avivace — CHANGES_REQUESTED** [review]

**Review by avivace — CHANGES_REQUESTED** [review]
Looks mostly okay!
Can you use `event:liksang2000-compo` as a tag?  You will also need to add it here: https://github.com/gbdev/database/blob/master/game-schema-d4.json#L114

File: entries/capman/game.json (hunk 1 above)
Comment (dag7dev) [reply entries/capman/game.json:1]: done!

**Comment by dag7dev** [comment]
> Looks mostly okay! Can you use `event:liksang2000-compo` as a tag? You will also need to add it here: https://github.com/gbdev/database/blob/master/game-schema-d4.json#L114

yup! Done

**Review by avivace — APPROVED** [review]

**Pull request merged**